packages/ftk-imager.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,21 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'FTK Imager'
+$category = 'Forensic'
+
+$exeUrl = 'https://d1kpmuwb7gvu1i.cloudfront.net/AccessData_FTK_Imager_4.7.1.exe'
```

Comment:
Will this URL be used when releasing a new version? I am thinking this because it doesn't include the version forth segment. If this is the case, an update will break the hash and consequently the package.

Reply:
I think the URL will change (even for minor version) but I'm not sure. However, I haven't any other solution and last update of this tools was 2 years ago (2022-01-19), so I don't think we have to worry about like other packages such as Chrome.